public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -128,3 +157,23 @@ export function toggleVizPanelLegend(vizPanel: VizPanel) {
 function hasLegendOptions(optionsWithLegend: unknown): optionsWithLegend is OptionsWithLegend {
   return optionsWithLegend != null && typeof optionsWithLegend === 'object' && 'legend' in optionsWithLegend;
 }
+
+function handleZoomOut(scene: DashboardScene) {
+  const timePicker = dashboardSceneGraph.getTimePicker(scene);
```

Comment:
@ivanortegaalba  - I'm adding this util to allow abstracting DashboardScene traversal to retrieve certain objects. I think you may find it useful for your General Settings work. 

FYI @torkelo  - think this util will be good to provide a way to make sure that structural changes in DashboardScene won't bring regressions in the future.